game/views.py

```
@@ -1,23 +1,43 @@
+
+from .forms import CustomUserCreationForm
+from django.views.decorators.csrf import ensure_csrf_cookie, csrf_exempt
 from django.views.decorators.http import require_GET, require_POST
+from django.contrib.auth.decorators import login_required
 
 from .engine import ChessGame
+from .models import GameResult
+
+logger = logging.getLogger(__name__)
+
+def _count_legal_moves(game):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_count_legal_moves`

(ANN202)